src/lt/macros.clj

```diff
@@ -47,30 +47,30 @@
     [(first body) (rest body)]
     [[] body]))
 
-(defmacro on [name & body]
+(defmacro ^:private on [name & body]
```

Comment:
Unless you really think these should be private / tagged as private shouldn't you use the :no-doc true meta tag https://github.com/weavejester/codox#metadata-options?